ca_on_ajax/people.py

```diff
@@ -2,48 +2,44 @@
 
 import re
 
-COUNCIL_PAGE = 'http://www.ajax.ca/en/insidetownhall/mayorcouncillors.asp'
+COUNCIL_PAGE = 'https://www.ajax.ca/en/inside-townhall/council-members.aspx'
 
 
 class AjaxPersonScraper(CanadianScraper):
     def scrape(self):
         page = self.lxmlize(COUNCIL_PAGE)
 
-        councillors = page.xpath('//ul[@class="subNav top"]/li/ul//li/a[contains(text(), "Councillor")] | //ul[@class="subNav top"]/li/ul//li/a[contains(text(), "Mayor")]')
-        assert len(councillors), 'No councillors found'
+        councillors = page.xpath('//table[@class="councilTable"]')
         for councillor in councillors:
-            name = councillor.text_content()
-
-            url = councillor.attrib['href']
-            page = self.lxmlize(url)
-
-            if councillor == councillors[0]:
+            image = councillor.xpath('.//@src')[0]
+            councillor_name = councillor.xpath('.//tr/td[1]/p[1]/img/@alt')
+            assert len(councillors), 'No councillors found'
+       
+            for i in councillor_name:
+                sep = '-'
+                name = i.split(sep, 1)[0]
+
+            if name == councillor_name[0]:
                 district = 'Ajax'
                 role = 'Mayor'
+
             else:
-                district = re.findall(r'Ward.*', page.xpath('//div[@id="printAreaContent"]//h1')[0].text_content())[0].strip()
-                role = page.xpath('//div[@id="printAreaContent"]//h1')[0].text_content()
-                role = re.search('((?:Regional )?Councillor)', role).group(1)
+                role=i.split(sep, 1)[-1]
+        
+            cell = councillor.xpath('.//p[contains(.,"Cel")]/text()')[0]
+            tel= councillor.xpath('.//p[contains(.,"Cel")]/text()')[1]
+            phone=cell.replace('\xa0', ' ')
+           
+
+            p = Person(primary_org='legislature', name=name, district='district', role='role')
```

Comment:
role and district and being set to the literal strings `'district'` and `'role'`.